README.md

```diff
@@ -1,33 +1,103 @@
-6. Open your browser and go to page http://chs.local/register-an-overseas-entity
+6. Open your browser and go to page <http://chs.local/register-an-overseas-entity>
+
+Environment variables used to configure this service in docker are located in the file `services/modules/overseas-entities/overseas-entities-web.docker-compose.yaml`
+
+### Requirements
+
+1. node v12.22.12 (Soon it will be v16)
+2. npm 6.14.16  (Soon it will be 8)
+3. Docker
+
+### Build and Test changes
+
+1. To compile the project use `make build`
+2. To test the project use `make test`
+3. or `make clean build test`
+
+### To build the Docker container
+
+1. `DOCKER_BUILDKIT=0 docker build --build-arg SSH_PRIVATE_KEY="$(cat ~/.ssh/id_rsa)" --build-arg SSH_PRIVATE_KEY_PASSPHRASE -t 169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/overseas-entities-web .`
+
+### Endpoints
+
+Method | Path | Description
+--- | --- | ---
+GET | `/register-an-overseas-entity` | Returns the landing page for the Register an overseas entity, starting point to different other pages to register an OE. URLs path [here](./src/routes/index.ts).
+GET | `/update-an-overseas-entity` | Returns the landing page for updating an overseas entity. URLs path [here](./src/routes/index.ts).
+GET | `/register-an-overseas-entity/healthcheck` | Returns responds with HTTP code `200` and a `OK` message body
+
+### Config variables (No feature flags)
+
+Key             |  Description               | Example Value
+----------------|--------------------------- |-------------------------
+ACCOUNT_URL | URL to account service | `http://account.url`
+API_URL | URL to API call | `http://api.url`
+CACHE_SERVER | Redis cache server | redis
+CDN_HOST | CDN host | cdn.host
+CHS_API_KEY | CHS API key for SDK call | key
+CHS_URL | CHS local url | `http://url`
+COOKIE_DOMAIN | The domain of the cookie | `http://url.local`
+COOKIE_NAME | The name of the cookie | __SID
+COOKIE_SECRET | The shared secret used in validating/calculating the session cookie signature | secret
+INTERNAL_API_URL | Internal API URL | `http://api.url`
+LANDING_PAGE_URL | Register OE landing Page | `/register-an-overseas-entity/sold-land-filter`
+UPDATE_LANDING_PAGE_URL | Update OE landing Page | `/update-an-overseas-entity/overseas-entity-query`
+LOG_LEVEL | LOG level | DEBUG
+OAUTH2_CLIENT_ID | OAUTH2 client ID | client ID
+OAUTH2_CLIENT_SECRET | OAUTH2 client secret | secret
+PAYMENT_FEE | Payment Fee | 100
+PIWIK_URL | Matomo URL | `http://url`
+PIWIK_SITE_ID | Matomo Site ID | 1
+PIWIK_START_GOAL_ID | Matomo Start goal ID | 2
+SHOW_SERVICE_OFFLINE_PAGE | Feature Flag | false
+VF01_FORM_DOWNLOAD_URL | Overseas entity verification checks statement URL | `https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/1095139/OE_VF01.pdf`
 
-These instructions are for a local docker environment.
+## Recommendations
 
+1. Use the [Visual Studio Code](https://code.visualstudio.com/) IDE for development.
```

Comment:
Could just say GoLand is another useful ide if you have the licence

Reply:
It seems GoLand does not have a [Community edition](https://www.jetbrains.com/help/go/faq-about-goland.html#is-there-a-goland-community-edition). Lets advocate open source  :)